MonoGame.Framework.Content.Pipeline/Builder/PipelineBuildEvent.cs

```diff
@@ -32,6 +33,11 @@ public PipelineBuildEvent()
         }
 
         /// <summary>
+        /// True if this item is a simple file copy, doing no importing or processing.
+        /// </summary>
+        public bool CopyOnly { get; set; }
```

Comment:
Instead of making this an additional property that has to be propagated thru the system.... make it implicit based on having an importer and/or processor defined.